doc/overview/datasets_index.rst

```diff
@@ -230,10 +230,6 @@ Each montage and block contains two marker types:
 - 1 = checkerboard reversal
 - 2 = same as 1 but for the first few trials (to keep separate if needed)
 
```

Comment:
I think this is from merging to master and we should adjust our PR so this isn't one of the files changed

Reply:
I did have a reference to the plot_80 tutorial here that I removed.